change/@ni-eslint-config-angular-9ad7619f-6545-4755-83fa-124156f09929.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Assuming this is draft? Don't think we are ready to merge for a while?

Reply:
It's a major version bump, so we can merge whenever we want, right?

Reply:
Makes releases trickier since main would be on that version. Changes to the v16 rules would have to be in a branch off main and we currently have frequent releases for updating a hard-coded list.

Reply:
For this repo the pipelines are setup to only release on `main` so branch releases are particularly not easy

Reply:
Makes sense. Also didn't realize how active this package is!

Reply:
I expect we will be able to pull the trigger on the update in about a week, so let's get this in.

Reply:
The merge and release is fast and I don't think significantly impacts testing, it's one additional tgz to test against. If that sounds like reasonable assertions I'd prefer to wait until the release train starts.